README.md

```diff
@@ -1,22 +1,25 @@
-# KerasNLP
+# KerasNLP: Modular NLP Workflows for Keras
 [![](https://github.com/keras-team/keras-nlp/workflows/Tests/badge.svg?branch=master)](https://github.com/keras-team/keras-nlp/actions?query=workflow%3ATests+branch%3Amaster)
 ![Python](https://img.shields.io/badge/python-v3.7.0+-success.svg)
 ![Tensorflow](https://img.shields.io/badge/tensorflow-v2.5.0+-success.svg)
 [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/keras-team/keras-nlp/issues)
 
-KerasNLP is a simple and powerful API for building Natural Language Processing
-(NLP) models within the Keras ecosystem.
 
-KerasNLP provides modular building blocks following
-standard Keras interfaces (layers, metrics) that allow you to quickly and
-flexibly iterate on your task. Engineers working in applied NLP can leverage the
-library to assemble training and inference pipelines that are both
-state-of-the-art and production-grade.
+KerasNLP is a natural language processing library that supports users through
+their entire development cycle. Our workflows are built from modular components 
+that have SoTA preset weights and architectures when used out-of-the-box and are
+easily customizable when more control is needed. We emphasize in-graph 
+computation for all workflows so that developers can expect easy 
+productionization using the Tensorflow ecosystem.
 
-KerasNLP can be understood as a horizontal extension of the Keras API —
-components are first-party Keras objects that are too specialized to be
-added to core Keras, but that receive the same level of polish as the rest of
-the Keras API.
+This library is an extension of the core Keras API; all high level modules are 
+`Layers` or `Models` that recieve that same level of polish as core Keras. If 
```

Comment:
Should we make these hyper links to https://keras.io/api/layers/ and https://keras.io/api/models/, in case we are getting newbies who need an introduction to the concepts? Or is that too much clutter.